linera-client/src/client_context.rs

```diff
@@ -207,7 +210,16 @@ where
             1 => format!("Client node for {:.8}", chain_ids[0]),
             n => format!("Client node for {:.8} and {} others", chain_ids[0], n - 1),
         };
-        let client = Client::new(node_provider, storage, 10, delivery, false, chain_ids, name);
+        let client = Client::new(
+            node_provider,
+            storage,
+            10,
+            delivery,
+            false,
+            chain_ids,
+            name,
+            NonZeroUsize::new(20).expect("Chain worker limit should not be zero"),
```

Comment:
I'm not sure I follow - there's a CLI option to set `max_loaded_chain` but chain worker limit is hardcoded here. Shouldn't we thread that value here?

Reply:
Ah, this is a test.